src/mappings/util.ts

```diff
@@ -165,21 +173,27 @@ export const getSigner = (extrinsic: SubstrateExtrinsic): string => {
 export const getDocValue = (
   doc: Codec
 ): Pick<AssetDocument, 'name' | 'link' | 'contentHash' | 'type' | 'filedAt'> => {
-  const {
-    uri: link,
-    content_hash: documentHash,
-    name,
-    doc_type: type,
-    filing_date: filedAt,
-  } = JSON.parse(doc.toString());
+  const document = JSON.parse(doc.toString());
+
+  const documentHash = extractValue(document, 'content_hash');
```

Comment:
I made the generic thinking of casting it to primitive types like number, string etc. Without explicitly specifying its as good as any. Since this was a JSON i left it this way